### DISCUSSION TITLE
V2ray for used new bing chat, why always error?

### DISCUSSION THREAD

**Discussion by wanqiangdehuoguo** [v2fly/v2ray-core#2437] [Q&A]
I just want use v2ray to use new bing chat in china. Unfortunatly, i always find the error just like "Sorry, looks like your network settings are preventing access to this feature."
Just want to discuss, it's microsoft's limitation or v2ray can't used for some prohibition？

**Comment by nursery01** [comment]
Because default cn route config does not proxy bing

Do you know "cn.bing.com"(中國特供版)? you opened is not "www.bing.com"

**Reply by wanqiangdehuoguo** [reply]
Thanks for answer, my aim is using v2ray as a proxy for bypassing "cn.bing.com" and use new bing chat.
And i think next explain is right "Microsoft may impose restrictions or geographic limitations on your account based on its previous activities"

**Comment by mydogshitgold** [comment]
V2Ray is just a tool that acts as a proxy. It doesn't mess with anything you're running, and has nothing to do with your Bing issue. While the proxy should be transparent to web applications, they may use WebRTC, DNS, history, etc to detect if a proxy is being used.

Microsoft may impose restrictions or geographic limitations on your account based on its previous activities, hence it is my recommendation to create a new account and use it exclusively through a proxy. Be sure to have a burner phone number that belongs to a place without restrictions in case they mark your account as violating and want to verify your phone number.

If you're using Edge (on Windows), which is prone to geo-locking even when using traditional proxy protocols like socks or HTTP, then you're better off using a transparent proxy and resetting this browser. Or use this [extension](https://github.com/anaclumos/bing-chat-for-all-browsers) with Chrome/Chromium or Firefox.

And don't forget to clear any cookies related to Microsoft or Bing when you are doing this.

**Reply by wanqiangdehuoguo** [reply]
Thanks a lot, your suggestion should take effect. I do understand why there is a network error. 

By the way, i just want to use new bing chat to improve work efficiency. If what you say is right, link new bing chat in China may be too much of a hassle. (T_T)

**Reply by mydogshitgold** [reply]
> improve work efficiency

I am happy that you have circumvented the imposed restrictions, but do exercise caution in placing undue reliance on Bing's results. GPT-4's ability to simulate human-like responses is noteworthy, but its susceptibility to [artificial hallucinations](https://en.wikipedia.org/wiki/Hallucination_(artificial_intelligence)) and arbitrary responses due to limited contextual awareness and **potential cognitive biases** is a source of factual error.

**Reply by nursery01** [reply]
Why your answer is like chat gpt

**Reply by nursery01** [reply]
Why your answer is like chat gpt

**Reply by nursery01** [reply]
Why your answer is like chat gpt

**Reply by nursery01** [reply]
Why your answer is like chat gpt

**Reply by mydogshitgold** [reply]
@nursery01

What is the reason for your shadow-like presence and repeated answers? I mean no offense but if you want have fun trolling my language, we can talk elsewhere instead generating spam mails for everyone watching this repo.

**Reply by nursery01** [reply]
Sorry, there is a network error here.

**Reply by mydogshitgold** [reply]
Fine, your discourse is truly captivating and enthralling to navigate and I find that very entertaining. Have a nice day, sir/madam.

**Reply by nursery01** [reply]
Noooooo, Don't angry,please

**Reply by mydogshitgold** [reply]
No one is mad. Please refrain from making noise. Feel free to incessantly bombard my email directly.
